docs/UserGuide.md

```diff
@@ -115,32 +140,7 @@ Examples:
 * `edit 2 n/Chocolate Bread t/` Edits the name of the 2nd item to be `Chocolate Bread` and clears all existing tags if
```

Comment:
Maybe it will be clearer to put it as "2nd item of the current list" as user may think that it is the overall list.